deploy/images/calico-node/apko.tmpl.yaml

```diff
@@ -1,11 +1,12 @@
+# source: https://github.com/chainguard-images/images/blob/4317e36114e989a8dcf744c4685cd60eb8582447/images/calico/configs/latest.node.apko.yaml
 contents:
   repositories:
     - https://packages.wolfi.dev/os
   keyring:
     - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
   packages:
     - busybox
-    - calico-node=__VERSION__
+    - calico-node=${VERSION}
```

Comment:
should this be PACKAGE_VERSION eventually?

Reply:
Yes